app/controllers/report_controller/widgets.rb

```diff
@@ -56,12 +56,12 @@ def widget_edit
       assert_privileges("widget_#{params[:id] ? "edit" : "new"}")
       id = params[:id] ? params[:id] : "new"
       return unless load_edit("widget_edit__#{id}", "replace_cell__explorer")
-      widget_get_form_vars
-      widget = @edit[:widget_id] ? MiqWidget.find_by_id(@edit[:widget_id]) : MiqWidget.new  # get the current record
-      widget_set_record_vars(widget)
-      if widget_validate_entries && widget.save_with_shortcuts(@edit[:new][:shortcuts].to_a)
-        AuditEvent.success(build_saved_audit(widget, @edit))
-        add_flash(_("%{model} \"%{name}\" was saved") % {:model => ui_lookup(:model => "MiqWidget"), :name => widget.title})
+      widget_get_form_vars # get current record(@widget) of MiqWidget
+      widget_set_record_vars(@widget)
+      if widget_validate_entries && @widget.save_with_shortcuts(@edit[:new][:shortcuts].to_a)
+        AuditEvent.success(build_saved_audit(@widget, @edit))
+        add_flash(_("%{model} \"%{name}\" was saved") % {:model => ui_lookup(:model => "MiqWidget"), :name =>
+                                                                                                     @widget.title})
```

Comment:
@lpichler I think it would be better to split the line on the parameter and move `:name => @widget.title` to the second line, lined up with `:model => ui_loo...` from the first line